rpc-interface/src/types.rs

```diff
@@ -653,21 +653,20 @@ pub enum Inherent {
 
 impl Inherent {
     pub fn try_from(hist_tx: HistoricTransaction) -> Option<Self> {
-        let hash = hist_tx.data.hash();
         Some(match hist_tx.data {
             HistoricTransactionData::Basic(_) => return None,
             HistoricTransactionData::Equivocation(_) => return None,
             HistoricTransactionData::Reward(RewardEvent {
-                validator_address,
-                reward_address,
+                ref validator_address,
+                ref reward_address,
                 value,
             }) => Inherent::Reward {
                 block_number: hist_tx.block_number,
                 block_time: hist_tx.block_time,
-                validator_address,
-                target: reward_address,
+                validator_address: validator_address.clone(),
+                target: reward_address.clone(),
```

Comment:
Why is cloning better than using `ref`?

Reply:
Wanted to move the `.tx_hash()` into the branch, but that requires the full object without moving stuff out. Since `Address` is just 20 bytes, cloning doesn't do much work here anyway.